generators/ci-cd/templates/_.gitlab-ci.yml

```diff
@@ -24,39 +22,59 @@ stages:
     - release
     - deploy
 
-<%_ if (buildTool == 'gradle') { _%>
+<%_ if (buildTool === 'gradle') { _%>
 before_script:
     - export GRADLE_USER_HOME=`pwd`/.gradle
+    - chmod +x gradlew
```

Comment:
In my workflow this is needed: https://github.com/jhipster/generator-jhipster/pull/5225#issuecomment-280451347